shared/src/helpers/date/compare-date/compare-date-without-time.ts

```diff
@@ -0,0 +1,9 @@
+function compareDatesWithoutTime(date1: Date, date2: Date): boolean {
+    return (
+        date1.getFullYear() === date2.getFullYear() &&
+        date1.getMonth() === date2.getMonth() &&
+        date1.getDate() === date2.getDate()
```

Comment:
Please, check if there is a simpler way to compare dates

Reply:
Done